src/rhsm/connection.py

```diff
@@ -297,7 +276,8 @@ def __init__(self, host, ssl_port=None,
         if no_proxy_override:
             os.environ['no_proxy'] = no_proxy_override
 
-        _fix_no_proxy()
+        utils.fix_no_proxy()
+        log.debug('Environment variable NO_PROXY=%s will be used' % no_proxy)
```

Comment:
Here should be:

log.debug('Environment variable NO_PROXY=%s will be used' % no_proxy_override)

Otherwise it prints wrong value.